speculos/mcu/seproxyhal.py

```diff
@@ -48,23 +48,61 @@ class SephTag(IntEnum):
 RENDER_METHOD = RenderMethods(0, 1)
 
 
-def ticker(add_tick):
-    """
-    Function ran in a thread to trigger a tick at a periodic interval.
+class TimeTickerDaemon(threading.Thread):
+    def __init__(self, add_tick, *args, **kwargs):
```

Comment:
```suggestion
    def __init__(self, add_tick: Callable, *args, **kwargs):
```
It's better to type-doc the function signature rather than the object properties as:
- `mypy` will not check the body of the method if the signature is not typed
- also `mypy` could flag unmatching  type when the method is called (`in super function, when instantiating TimeTickerDaemon, the constructor was expecting A but got B`) rather than inside the method itself (`in inner method __init__, property add_tick was expecting A but got B `), which is generally more helpful
- I _think_ most doc tools (such as doxygen) can extract argument types from signature

Reply:
Done